python/experimental/cudaq/kernel/ast_bridge.py

```diff
@@ -102,39 +102,39 @@ def isMeasureResultType(self, ty):
 
     def getIntegerType(self, width=64):
         """
-        Return an MLIR IntegerType of the given bit width (defaults to 64 bits).
+        Return an MLIR integer type of the given bit width (defaults to 64 bits).
```

Comment:
IntegerType (one word) is intended. I think you should wrap with the tick marks instead (and everywhere else it' written)

Reply:
updated